fold-this.el

```diff
@@ -294,7 +300,7 @@ If narrowing is active, only in it."
 (defun fold-this--save-overlay-to-alist (overlay)
   "Add an OVERLAY position pair to the alist."
   (when (eq (overlay-get overlay 'type) 'fold-this)
-    (let* ((pos (cons (1- (overlay-start overlay)) (1+ (overlay-end overlay))))
+    (let* ((pos (cons (- (overlay-start overlay) fold-this-skip-chars) (+ (overlay-end overlay) fold-this-skip-chars)))
```

Comment:
This line is a bit too long, I would wrap it as so

```suggestion
    (let* ((pos (cons 
                 (- (overlay-start overlay) fold-this-skip-chars)
                 (+ (overlay-end overlay) fold-this-skip-chars)))
```

Also makes it easier to see what the arguments to `cons` are.

Reply:
Fixed in commit 57d2696